subprojects/file-watching/src/main/java/org/gradle/internal/watch/registry/impl/HierarchicalFileWatcherUpdater.java

```diff
@@ -121,5 +115,6 @@ private void updateWatchedDirectories(Set<Path> newWatchRoots) {
             );
             watchedRoots.addAll(newWatchRoots);
         }
+        LOGGER.info("Watching {} directory hierarchies to track changes", watchedRoots.size());
```

Comment:
Not belonging in this PR, but this message would be very useful to see in general. Shall we promote it to `warn`?

Reply:
We can. Shall I do this as part of the current PR?

Reply:
Up to you.

Reply:
Let's do it in a different PR, I don't want to fix the logging tests now.